.gitignore

```diff
@@ -31,14 +31,21 @@
 # Visual Studio 15+ auto generated files
 Generated\ Files/
 
+# Backup folder for Package Reference Convert tool in Visual Studio 2017
+MigrationBackup/
```

Comment:
Don't think we need this, we really don't care about Visual Studio 2017, it can't build the repo at all anyway. I know this was here before and it was just moved, but we might just delete it entirely at this point.

Reply:
The entire git ignore is being updated from upstream and not really specific to this repo except some entries. Even if we delete it here, it would still show up whenever there's an update to those files.

Reply:
> "The entire git ignore is being updated from upstream"

What do you mean? What upstream?

Reply:
We have an internal fork of the attributes and ignores that's gets updated every once in a while by our dev teams. The one we have is more focused than the general ignores template available on GitHub. That is the upstream. And its also where I got the git attributes, editorconfig files.